src/instructlab/model/serve.py

```diff
@@ -64,6 +64,20 @@
         " Example: --vllm-args='--dtype=auto' --vllm-args='--enable-lora'"
     ),
 )
+@click.option(
+    "--host",
+    type=str,
+    default=None,
+    show_default=True,
+    help="The host address to bind to.",
+)
+@click.option(
+    "--port",
+    type=click.INT,
+    default=None,
+    show_default=True,
+    help="The port number to bind to.",
+)
```

Comment:
I recommend that you add an option `--host-port` instead. It will automatically get its default value from `config.serve.host_port`.

```suggestion
@click.option(
    "--host-port",
    type=str,
    required=True,  # default from config
    help="Listen host:port for the server (example: 'localhost:8080').",
)
```